python/hidet/graph/ops/matmul/resolve.py

```diff
@@ -96,36 +97,45 @@ class MatmulResolveRule(ResolveRule):
     This resolve rule also parallelize k dimension when possible, and determine the mma instruction.
     """
 
-    def run_batch_matmul(self, a: Tensor, b: Tensor) -> Tensor:
-        parallel_k = self.get_config('parallel_k', default='default')  # 'default', 'search', 2, 4, ...
-        mma = self.get_config('mma', default='simt')  # 'simt', 'mma'
-
-        if any(not isinstance(v, int) for v in a.shape + b.shape):
-            nparts = 1
+    def run_batch_matmul(self, a: Tensor, b: Tensor, is_cpu: bool) -> Tensor:
+        if is_cpu:
```

Comment:
We can directly check the device of tensor, without the need to pass `is_cpu` as parameter.
```
is_cpu = a.device.is_cpu()
```